platform-db-core/src/main/java/com/softicar/platform/db/core/connection/DbConnection.java

```diff
@@ -248,10 +249,19 @@ public int executeInsert(DbStatement statement) {
 		}
 	}
 
-	private boolean isFirstColumnAutoIncrement(ResultSet resultSet) throws SQLException {
+	private boolean hasNumericFirstColumn(ResultSet resultSet) throws SQLException {
 
 		var metaData = resultSet.getMetaData();
-		return metaData.getColumnCount() > 0 && metaData.isAutoIncrement(1);
+		if (metaData.getColumnCount() > 0) {
+			switch (metaData.getColumnType(1)) {
+			case Types.BIGINT:
```

Comment:
hmm, in line 238, we call `getInt` which will truncate the BIGINT...maybe we should add some TODO or FIXME in 238, referring to our missing BIGINT-auto-increment support?